src/main/java/org/folio/rest/service/report/RefundReportService.java

```diff
@@ -464,11 +465,11 @@ private static String singleOrDefaultMessage(List<String> values, String message
   }
 
   private static String getStaffInfo(String comments) {
```

Comment:
What happens if we just return `null` from this method? Will it break anything? I'm just thinking if we really need this `defaultString` call here...

Reply:
It will break RefundReportEntryMatcher and will remove staffInfo and patronInfo fields from json for ReportEntry